templates/apollo-graphql-react/files/.platform/services.yaml

```diff
@@ -0,0 +1,8 @@
+# The services of the project.
+#
+# Each service listed will be deployed
+# to power your Platform.sh project.
+
+db:
+  type: mongodb:3.6
+  disk: 512
```

Comment:
Mongo is very disk hungry.  We may want to give it a full gig out of the box, so it has room to grow.  It uses like 300 MB itself just to start up. :-/

Reply:
Done in 712ae94ecda31b7fac66dabce16241644a7df1cc